config/config-example.inc.php

```diff
@@ -2,6 +2,8 @@
 
 mb_internal_encoding('UTF-8');
 
+$routes = json_decode(file_get_contents('routes.json'));
```

Comment:
Heh, I was trying to think of a way to get them to only need one place. This is actually a pretty good approach.

Reply:
I agree, this is pretty clever! Nice, Annika :)

Reply:
LOL, it turns out I suggested this earlier and forgot:

https://github.com/smogon/pokemon-showdown-client/pull/1543#discussion_r449928418

Reply:
Clever Zarel then!